its/src/test/java/com/sonar/it/shared/ScannerCliTest.java

```diff
@@ -56,23 +57,59 @@ public void init() {
   }
 
   @Test
-  public void scannerCliWithRazorPages() {
+  public void scannerCliWithRazorPagesMainCode() {
```

Comment:
I think we should improve a bit the test name. Just by reading it we can't figure it out what is supposed to test.
WDYT about `givenRazorPagesMainCode_whenScannerForCliIsUsed_displaysWarning` or something similar.

I think we should reconsider a bit also the other names.